```
{ ScriptError 
staging/src/k8s.io/component-base/metrics/prometheus/slis/metrics.go:22:2: ST1019: package "k8s.io/component-base/metrics" is being imported more than once (stylecheck)
	compbasemetrics "k8s.io/component-base/metrics"
	^
staging/src/k8s.io/component-base/metrics/prometheus/slis/metrics.go:23:2: ST1019(related information): other import of "k8s.io/component-base/metrics" (stylecheck)
	k8smetrics "k8s.io/component-base/metrics"
	^
}
```

CI failure needs to be fixed.

/kind feature
/lgtm cancel

--- a/staging/src/k8s.io/component-base/metrics/prometheus/slis/metrics.go
+++ b/staging/src/k8s.io/component-base/metrics/prometheus/slis/metrics.go
@@ -55,6 +55,7 @@ var (
 func Register(registry k8smetrics.KubeRegistry) {
 	registry.Register(healthcheck)
 	registry.Register(healthchecksTotal)
+	_ = k8smetrics.RegisterProcessStartTime(registry.Register)
 }
 
 func ResetHealthMetrics() {

